Bixal/move.mil
# Feature/icon fixes

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…
Contains hotfixes for the Weight calc json tool icons.

## Testing

1. Pull changes.
2. go to http://move.mil.localhost:8000/parser/weight_calculator
3. Check icon:

## Screenshots

_Attach relevant before and after screenshots here._
